src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Windows.cs

```diff
@@ -14,8 +15,8 @@ internal static class NameResolutionPal
         // used by GetHostName() to preallocate a buffer for the call to gethostname.
         //
         private const int HostNameBufferLength = 256;
-        private static volatile bool s_Initialized;
-        private static object s_InitializedLock = new object();
+        private static bool s_initialized;
```

Comment:
@pgavlin The https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md isn't clear on this: should we not capitalize after s_?